### ISSUE TITLE
Build enviroment

### ISSUE THREAD

**Issue by dv8inpp** [Blup1980/microsip_42#2]
Can you provide detail of your build environment.
Windows version
Visual Studio Version
Pjsip version
opus version

thanks

**Comment by MehdiRaza1998** [comment]
I need the details of your environment too.

**Comment by Meulie007** [comment]
@dv8inpp 
@MehdiRaza1998 

Did you find an answer for your question?

**Comment by dv8inpp** [comment]
No, I gave up as Microsip didn't provide presence information or something
I needed.
